Backend/Backend/Services/EpisodeService.cs

```diff
@@ -22,12 +28,32 @@ public class EpisodeService : IEpisodeService
 
         }
 
+        public double GetEpisodeDuration(string path)
+        {
+            try
+            {
+                Console.WriteLine(path);
```

Comment:
you, uhh, should not be using console.WriteLine for this. This only outputs while we are running in debug mode, and doesnt really help us at all right now. I guess this is placeholder for logging?

Reply:
yeah I think you just left debugging info in here. Should be ok to remove it